volatility3/framework/plugins/windows/skeleton_key_check.py

```diff
@@ -292,7 +333,8 @@ def _get_rip_relative_target(self, inst) -> int:
         return inst.address + inst.size + opnd.mem.disp
 
     def _analyze_cdlocatecsystem(self, function_bytes: bytes,
-                                       function_start: int, 
+                                       function_start: int,
+                                       cryptdll_types: interfaces.context.ModuleInterface,
                                        proc_layer_name: str) -> Tuple[int, int]:
```

Comment:
Looks like the return type needs updating to something like `Optional[interfaces.objects.ObjectInterface]`?